src/sentry/issues/constants.py

```diff
@@ -12,11 +20,24 @@
 }
 
 
-def get_issue_tsdb_group_model(issue_category: GroupCategory) -> TSDBModel:
+def get_issue_tsdb_group_model(issue_category: GroupCategory, project: Project) -> TSDBModel:
+    # TODO: Remove this entire branch when we remove the option, and remove `PERFORMANCE` from
+    # ISSUE_TSDB_GROUP_MODELS
+    if issue_category == GroupCategory.PERFORMANCE and issue_category_can_create_group(
+        GroupCategory.PERFORMANCE, project
+    ):
+        return tsdb.models.group_generic
+    # TODO: Need to get generic when perf search issue flag is true and perf issue
     return ISSUE_TSDB_GROUP_MODELS.get(issue_category, tsdb.models.group_generic)
 
 
-def get_issue_tsdb_user_group_model(issue_category: GroupCategory) -> TSDBModel:
+def get_issue_tsdb_user_group_model(issue_category: GroupCategory, project: Project) -> TSDBModel:
+    # TODO: Remove this entire branch when we remove the option, and remove `PERFORMANCE` from
+    # ISSUE_TSDB_USER_GROUP_MODELS
+    if issue_category == GroupCategory.PERFORMANCE and issue_category_can_create_group(
```

Comment:
Couldn't use the feature flag here since a lot of this code is called as part of post process group